cmake.config/config.h.in

```diff
@@ -54,6 +54,8 @@
 #  undef HAVE_SYS_UIO_H
 # endif
 #endif
+#cmakedefine HAVE_DIRFD
+#cmakedefine HAVE_FLOCK
```

Comment:
I suppose libuv doesn't have an interface for these?

Reply:
I searched libuv's repository and cannot find any call to `flock() or dirfd()`.